suites/create.js

```diff
@@ -308,6 +308,101 @@ export function runDataIntegrityProofFormatTests({
           }
         }
       });
+      describe('Algorithms', function() {
+        describe('Add Proof', function() {
+          it('Whenever this algorithm encodes strings, ' +
+            'it MUST use UTF-8 encoding.',
+          function() {
+            for(const proof of proofs) {
+              chai.expect(proof.proofValue.isWellFormed()).to.be.true;
```

Comment:
I'd import `expect` so we don't see `chai`everywhere.

Reply:
I've imported expect

Reply:
I've imported expect